src/olympia/lib/crypto/tests/test_packaged.py

```diff
@@ -274,8 +274,11 @@ def test_call_signing(self):
         assert (
             'name="file"; filename="mozilla.sf"\r\n\r\n'
             'Signature-Version: 1.0\n'
-            'MD5-Digest-Manifest: UrEJ9n5q8I9UW2KlFUJDkA==\n'
-            'SHA1-Digest-Manifest: lTdbRmVMF7o/C+BT9GnMQne2Ap4=') in call.body
+            'MD5-Digest-Manifest: 38vYqmQVrnRoU0Ac00upqw==\n'
+            'SHA1-Digest-Manifest: 5zn5SCe3RDBgGhSCK8rFJi98JCw=\n'
+            'SHA256-Digest-Manifest: '
+                '4ZpVcLE00kZymr5C4M/KYTat9tj5kncqtv84gvlbT5g='  # noqa
```

Comment:
single quote after `SHA256-Digest-Manifest: ` looks like a typo, no?

Reply:
It's not, the string will directly continue on the next line. It's my way of staying in our 79 character line-length limit :)

Reply:
TIL :)